README.md

```diff
@@ -44,6 +45,8 @@ All the links you'll need are listed below. If you'd like to contribute to the P
 
 - [Consensus Changelog](consensus/CHANGELOG.md)
 - [Utility Changelog](utility/CHANGELOG.md)
+- _Coming Soon: P2P Changelog_
+- _Coming Soon: Persistence Changelog_
```

Comment:
🤔 We should really make sure merges don't go without changing these files. Maybe time for some pre commit tooling to validate this?

Reply:
We can just ping each other when we find this.

My answer would be sure as long as it doesn't detract from our day-to-day responsibilities. Even better, create a Github issue for it as a starter task for someone in the community.

Reply:
Kudos for the suggestion, I will create an issue for this.